Makefile

```diff
@@ -35,6 +35,7 @@ LDFLAGS=\
 	-s USE_FREETYPE=1 \
 	-s USE_LIBPNG=1 \
 	-std=c++14 \
+  -L/usr/local/lib -lsqlite3 \
```

Comment:
We definitely shouldn't ever pull in things from a system directory like this -- that could be anything.

Is this line even necessary?  Can we instead try to link sqlite only to the Python sqlite extension and avoid putting it in the main wasm build?

Reply:
Yeah I wasn't sure how to link sqlite just to the extension, because it seems the way the CPython build process is working is it takes all of the files `_sqlite/*.c` and creates `.o` files for each of them and then all of the `.o` files get linked in `libpython3.7.a`.

Reply:
Here's how `libpython3.7.a` gets built:

```
/src/emsdk/emsdk/emscripten/tag-1.38.22/emar rcs libpython3.7.a Modules/getbuildinfo.o Parser/acceler.o Parser/grammar1.o ...
```

I tried manually (from the CLI) appending both `/src/cpython/build/sqlite-autoconf-3270200/.libs/libsqlite3.la` and `/src/cpython/build/sqlite-autoconf-3270200/.libs/libsqlite3.a` to the end of that long `ar` command, but the sqlite symbols still show up as undefined:

```
root@1d971ca4efbe:/src/cpython/build/3.7.0/Python-3.7.0# llvm-nm libpython3.7.a | grep sqlite3_libversion
         U sqlite3_libversion_number
         U sqlite3_libversion
```

Any ideas?

Reply:
Ah I have a feeling that you can't add a `.a` to another `.a` and that I have to add the individual `.o` files -- e.g.: I should add to that link line:

```
/src/cpython/build/sqlite-autoconf-3270200/*.o
```

Reply:
That seems potentially fruitful to try.